src/status_im2/contexts/onboarding/create_password/view.cljs

```diff
@@ -1,42 +1,197 @@
-   [quo/page-nav
-    {:align-mid?            true
-     :mid-section           {:type :text-only :main-text ""}
-     :left-section          {:type                :blur-bg
-                             :icon                :i/arrow-left
-                             :icon-override-theme :dark
-                             :on-press            #(rf/dispatch [:navigate-back])}
-     :right-section-buttons [{:type                :blur-bg
-                              :icon                :i/info
-                              :icon-override-theme :dark
-                              :on-press            #(js/alert "Pending")}]}]])
-
-(defn page
+  [rn/image
+   {:style       style/image-background
+    :blur-radius 13
+    :source      (resources/get-image :onboarding-blur-bg)}])
+
+(defn header
   []
-  [rn/view {:style style/page-container}
-   [navigation-bar]
-   [rn/view {:style {:padding-horizontal 20}}
-    [quo/text
-     {:size   :heading-1
-      :weight :semi-bold
-      :style  {:color colors/white}} "Create profile password"]
-    [quo/button
-     {:on-press #(rf/dispatch [:navigate-to :enable-biometrics])
-      :style    {}} (i18n/label :t/continue)]]])
+  [rn/view {:style style/heading}
+   [quo/text
+    {:style  style/heading-title
+     :weight :semi-bold
+     :size   :heading-1}
+    (i18n/label :t/password-creation-title)]
+   [quo/text
+    {:style  style/heading-subtitle
+     :weight :regular
+     :size   :paragraph-1}
+    (i18n/label :t/password-creation-subtitle)]])
+
+(defn password-with-hint
+  [{{:keys [text status shown]} :hint :as input-props}]
+  [rn/view
+   [quo/input
+    (-> input-props
+        (dissoc :hint)
+        (assoc :type           :password
+               :override-theme :dark
+               :blur?          true))]
+   [rn/view {:style style/label-container}
+    (when shown
+      [:<>
+       [quo/icon (if (= status :success) :i/check-circle :i/info)
+        {:container-style style/label-icon
+         :color           (style/label-icon-color status)
+         :size            16}]
+       [quo/text
+        {:style (style/label-color status)
+         :size  :paragraph-2}
+        text]])]])
+
+(defn password-inputs
+  [{:keys [passwords-match? on-change-password on-change-repeat-password on-input-focus
+           password-long-enough? empty-password? same-password-lenght?]}]
+  (reagent/with-let [show-hint-2? (reagent/atom false)]
+    (let [_ (when same-password-lenght? (reset! show-hint-2? true))
+          hint-1-status (if password-long-enough? :success :neutral)
+          hint-2-status (if passwords-match? :success :danger)
+          hint-2-text   (if passwords-match?
+                          (i18n/label :t/password-creation-match)
+                          (i18n/label :t/password-creation-dont-match))
+          error?        (and @show-hint-2?
+                             (not passwords-match?)
+                             (not empty-password?))]
+      [rn/view
```

Comment:
You could use `[:<>`

Reply:
Hey @briansztamfater 
AFAIK, a React.Fragment is not the same than a view component with empty props.

Previously someone else told me the same, but a fragment doesn't add a node:
https://legacy.reactjs.org/docs/fragments.html

or am I missing something that is different in React Native?

Reply:
You are right, in fact, I am not sure if we need this extra node in the DOM, so if the purpose of using a view without style is just grouping children components, maybe a Fragment is a better option. Let me know if that make sense to you.

Reply:
I've checked it, and in this case it can be safetly replaced, I did it.

The main problem is when you have something like this:

``` clojure
[rn/view {:style {:justify-content :space-between}}
 [component-1]
 [component-2]
 [component-3]]
 ```
 
 and if component-2 looks like this:
 ```clojure
 [rn/view
 [component-2-1]
 [component-2-2]]
 ```
 If we replace component-2's view with `:<>` things won't work as expected becuse of the `:justify-content`'s value on its parent. 
 
 In this case, the parent is only adding some margins, so it's safe to remove the node. Thanks for the suggestion :+1: